packages/@react-spectrum/provider/stories/Provider.stories.tsx

```diff
@@ -164,7 +168,103 @@ storiesOf('Provider', module)
             <Breakpoint />
           </Provider>
         );
-      });
+      })
+      .add(
+        'breakpoints updated',
+        () => (
+          <ProviderWithBreakpointsEditor />
+          ));
```

Comment:
extra 2 spaces
```suggestion
        ));
```

Reply:
Good catch. Has there been discussion about enforcing any style guidelines for the repository? I ended up cleaning up a few other (cascading) indentation issues in this code (with this fix).

Reply:
We generally have pretty strict linting, this is one that (generally) all of our IDE's are configured for, so I think we've just forgotten to include it